### PR TITLE
Added navigation bar #31

### LINKED ISSUE #31: Add a navbar that's persistent
- Create a component for the navbar
- Make it persistent on all pages

The navbar component should be placed in App.tsx

## Steps
- Create a sitelayout component which contains the navbar (example: https://github.com/GryPr/next-experiments/blob/master/src/components/SiteLayout/SiteLayout.tsx)
- Use sitelayout in the app.tsx (example: https://github.com/GryPr/next-experiments/blob/master/src/pages/_app.tsx)

image

### PR DESCRIPTION
**Pull request by salehatrq**
Created navigation bar component
Closes #31

### CODE REVIEW

File: frontend/src/Components/SiteLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import './SiteLayout.css';
+import SearchIcon from '@material-ui/icons/Search'
```
Comment (GryPr) [inline frontend/src/Components/SiteLayout.tsx:2]: PR check seems to produce the following error, so we can't merge the PR until it's fixed (or else the deployment won't work)

```
./src/Components/SiteLayout.tsx
Cannot find module: '@material-ui/icons/Search'. Make sure this package is installed.

You can install this package by running: npm install @material-ui/icons/Search.
```

**Comment by AliZ786** [comment]
Looks good overall, very good job!

**Review by GryPr — CHANGES_REQUESTED** [review]
Need to fix dependency in the project before merging.

**Review by GryPr — APPROVED** [review]
Everything's good!

**Review by bhattirahat — APPROVED** [review]
good work

**Pull request merged**